client/src/components/MobilePaginationDotsContainer.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { FaCircle } from 'react-icons/fa';
+import '../../dist/MobilePaginationDotsContainer.css';
+
+const MobilePaginationDotsContainer = (props) => {
+  let dots = props.photo.map((photo, i) => {
+    const photoClassName = i === props.currentPhoto ? "activeDot" : "dot";
+      return <div key={i} className={photoClassName}>
+        <FaCircle /> </div>
```

Comment:
If we break this into a multi-line set of HTML that we're returning, we should wrap the returned HTML/JSX in parens:

```
return (
  <div>
    <otherThing />
  </div>
);